example web components

jsaikiran19 opened the pull request.

bherr2 commented:
Do these work?

bherr2 commented:
> > Do these work?
> 
> These are just example htmls bruce. It won't load the application. It's just for reference on how to use the web component html.

If they don't work, they are _terrible_ examples.

jsaikiran19 commented:
> > > Do these work?
> > 
> > 
> > These are just example htmls bruce. It won't load the application. It's just for reference on how to use the web component html.
> 
> If they don't work, they are _terrible_ examples.

Yeah that was the initial draft for those htmls. I don't think I've updated them. I just updated the ftu-large and ftu-small htmls with the latest changes.
